docs/content/integration/openid-connect/portainer/index.md

```diff
@@ -88,6 +89,7 @@ To configure [Portainer] to utilize Authelia as an [OpenID Connect 1.0] Provider
    9. Redirect URL: `https://portainer.{{< sitevar name="domain" nojs="example.com" >}}`
    10. User Identifier: `preferred_username`
    11. Scopes: `openid profile groups email`
+   12. Token Endpoint Auth Method: `client_secret_post`
```

Comment:
Please show the screenshot of this in Portainer using the example values.

Reply:
Actually, that was a mistake! This isn't a portainer setting. Well, at least not directly. 

It looks like [they recently added](https://github.com/portainer/portainer/commit/ffc66647f867393f6274dd747c47494eb54f53f1#diff-6c81c4be8002000c7704490af244731ad3de1e41e161a26606e95c2ef1b11bcaR91) a new "Auth Style" setting that allows users to select "In Headers" or "In Params" which toggles between `client_secret_basic` and `client_secret_post` respectively. Since the default is "Auto Detect" and it defaulted to "In Params" I've updated this checklist and screenshot accordingly.

Reply:
I think we should set it to `In Params` explicitly. Auto Detection mechanisms can cause a lot of confusion for users when debugging as they can produce multiple errors rather than single errors.

Reply:
great callout. updated!